x-pack/test/security_solution_cypress/cypress/e2e/explore/inspect/inspect_button.cy.ts

```diff
@@ -21,72 +21,66 @@ import { visitWithTimeRange } from '../../../tasks/navigation';
 import { waitForWelcomePanelToBeLoaded } from '../../../tasks/common';
 import { selectDataView } from '../../../tasks/sourcerer';
 import { postDataView } from '../../../tasks/api_calls/common';
+import { mockRiskEngineEnabled } from '../../../tasks/entity_analytics';
 
 const DATA_VIEW = 'auditbeat-*';
 
-// FLAKY: https://github.com/elastic/kibana/issues/178367
-describe.skip(
-  'Inspect Explore pages',
-  { tags: ['@ess', '@serverless', '@skipInServerlessMKI'] },
-  () => {
-    before(() => {
-      // illegal_argument_exception: unknown setting [index.lifecycle.name]
-      cy.task('esArchiverLoad', { archiveName: 'risk_users' });
-      cy.task('esArchiverLoad', { archiveName: 'risk_hosts' });
+describe('Inspect Explore pages', { tags: ['@ess', '@serverless'] }, () => {
```

Comment:
Have you checked that this test works on MKI as well?

Reply:
Yep, looks like the explore tests pass on MKI :) Thanks for making the test run: https://buildkite.com/elastic/kibana-serverless-security-solution-quality-gate-explore/builds/1560 👍🏾